Furball.Vixie.TestApplication/Tests/TestNewRenderer.cs

```diff
@@ -107,7 +107,7 @@ public unsafe class TestNewRenderer : Screen {
     }
 
     public override void Draw(double deltaTime) {
-        GraphicsBackend.Current.Clear();
+        TestGame.Instance.WindowManager.GraphicsBackend.Clear();
```

Comment:
Why has this been only changed instead of removed entirely like in other places?

Reply:
> Why has this been only changed instead of removed entirely like in other places?

because i have a mental illness